### PR TITLE
Remove checks

### PR DESCRIPTION
**Pull request by timschwab**
Removed the unnecessary parts of the JSON objects: "checks" and "lines."

### CODE REVIEW

**Comment by timschwab** [comment]
Nah, I can take care of it real quick.

File: right/javascript/vc.js [hunk 1]
```diff
@@ -7,23 +7,31 @@ function submitAnswer() {
...
+    else
+    {
+        $("#dialog-message").html("Sorry, not the intended answer. Try again!");
         $("#dialog-box").dialog("open");
         verifying = false;
         $("#right .footetteDisabled").attr("class", "footette");
-            
-        createEditor.focus();
-        return;
+        createEditor.focus();
```
Comment (timschwab) [inline right/javascript/vc.js]: I'm not sure why it is saying that createEditor is not defined

**Comment by yushan87** [comment]
Line endings...

**Comment by timschwab** [comment]
I know! I keep thinking I'm getting them by deleting the end and then putting them back, but I can't tell until I commit. I'm not sure how they got in there in the first place.

**Pull request merged**